environments/demo/demo-platform-hmcts-net.yml

```diff
@@ -58,3 +58,9 @@ cname:
   - name: "adoption-web"
     ttl: 300
     record: "hmcts-demo.azurefd.net"
+  - name: "afdverify.civil-citizen-ui"
+    ttl: 300
+    record: "afdverify.hmcts-demo.azurefd.net"
+  - name: "civil-citizen-ui"
```

Comment:
@mounikahmcts just noticed the sample PR in HMCTS ways of working has a `-aks` suffix in the name, not sure if that has any incidence

Reply:
I think we don't need to have aks here as per documentation 
https://hmcts.github.io/ways-of-working/path-to-live/public-dns.html#setup-required-for-an-application

https://github.com/hmcts/azure-platform-terraform/blob/fa598ca88221555abcbe54a52f0738ca20d48f59/environments/prod.tfvars#L17